### DISCUSSION TITLE
App in other languages: Using translated version of SpotiFlyer

### DISCUSSION THREAD

**Discussion by ElizabethWega** [Shabinder/SpotiFlyer#530] [Q&A]
I know that there are many languages added to SpotiFlyer, as to be easer to other people speaking it. I was wondering how do I even change the language of the app? Thanks in advance for answering. :)

**Comment by Shabinder** [comment]
As of now app uses your locale to choose a language to load,

To Change you need to change system language and probably restart so that apps can detect the new locale and show respective language

**Reply by ElizabethWega** [reply]
I already use my language as the system's. Specifically, I'm using Croatian, but in our language it is translated into Hrvatski. May that be the cause why it's not recognising it?

**Reply by Shabinder** [reply]
I m not following, can you specify which Language is set and which file from translations folder should be active ideally according to you ?

**Reply by ElizabethWega** [reply]
The language set in my phone is Croatian. The file that should be active from translations folder is Strings_cro.properties

**Reply by ElizabethWega** [reply]
UPDATE: As I have my desired language activated in my system, I restarted my phone. Nothing. Remained in English. Maybe I need to change the prefix in strings from cro to another one?

**Reply by Shabinder** [reply]
> 
> 
> UPDATE: As I have my desired language activated in my system, I restarted my phone. Nothing. Remained in English. Maybe I need to change the prefix in strings from cro to another one?

Yeah I guess so the suffix in Strings_cro is most probably the issue here, will see the proper suffix for Croatian and will edit that

**Reply by ElizabethWega** [reply]
Thank you! I appreciate that! :)
